src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+import './calculator.css';
+
+export default function App() {
```

Comment:
- [ ] Kidnly consider renaming this component to `Calculator` and consider adding the deleted App component and importing the `Calculator`  component on it.